plugin/pkg/shoot/oidc/openidconnectpreset/admission_test.go

```diff
@@ -31,7 +31,7 @@ var _ = Describe("OpenID Connect Preset", func() {
 
 		BeforeEach(func() {
 			namespace := "my-namespace"
-			shootName := "shoot"
+			shootName := "shoot0"
```

Comment:
No need to change the name
```suggestion
			shootName := "shoot"
```